smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/HttpRpcProtocolGenerator.java

```diff
@@ -311,10 +321,16 @@ private void readResponseBody(GenerationContext context, OperationShape operatio
      * Writes the code that loads an {@code errorCode} String with the content used
      * to dispatch errors to specific serializers.
      *
-     * <p>Three variables will be in scope:
+     * <p>Two variables will be in scope:
      *   <ul>
-     *       <li>{@code output}: a value of the HttpResponse type.</li>
-     *       <li>{@code data}: the contents of the response body.</li>
+     *       <li>{@code errorOutput} or {@code parsedOutput}: a value of the HttpResponse type.
```

Comment:
Does this require changes to the [function generated in `generateErrorDeserializer`](https://github.com/awslabs/smithy-typescript/pull/66/files#diff-1bdb59e7660a9ee24a5655eb86b8383dR286) if the raw `HttpResponse` would be returned? It seems like that is expecting a parsed response when invoking the shape deserializer.

Reply:
This only applies to  `writeErrorParser()`. But I still need to change that variable name there to `parsedOutput` tho it won't break the code